docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst

```diff
@@ -0,0 +1,53 @@
+ 
+I went through the previous year's accepted organizations list and shortlisted 2-3 organizations. I started contributing to a few projects based on android and kotlin, but I lost interest after some time.
+It was about December, and I hadn't chosen any organization yet.
+
+I heard about FURY from one of my seniors. I started looking at its docs. Picked up some books from the library to brush up on my concepts of computer graphics. The documentation of FURY is remarkable. I created my `first pull request <https://github.com/fury-gl/fury/pull/520>`_ by improving one of the tutorials, and It got merged in a few days. 
+
+I started looking at the source code of FURY and tried to understand it again. I read the `API reference` part of FURY docs this time along with the documentation of `VTK`. I also tried to solve some of the open issues. I created a few pull requests for the same. My first contribution was in the fury `primitive` class, I created a sphere primitive (inspired by the sphere in Blender).
+
+I started looking at bugs and improvements that can be made to the source code and created PRs for the same. During this time I was also learning how computer graphics work from the `UCSC lectures <https://www.youtube.com/channel/UCSynd9Z5RdIpKfvTCITV_8A/videos>`_  & `OpenGL by Victor Gordon <https://youtube.com/playlist?list=PLPaoO-vpZnumdcb4tZc4x5Q-v7CkrQ6M->`_.
+
+After the accepted organizations were announced, I was so happy that FURY got selected this year for GSoC and went straight to the Ideas list. The first project idea was `glTF Integration`. I heard about the `glTF` file format before but didn't know how it works. So I went straight through the reference materials provided on the `wiki <https://github.com/fury-gl/fury/wiki/Google-Summer-of-Code-2022-(GSOC2022)>`_. I read a lot of documentation by the Khronos group. I also tried to implement a basic glTF loader during this time using VTK's built-in `glTFReader`.
+
+I also liked the Improve UI drawing project idea (I had a basic understanding of line drawing algorithms and rasterizations at that time) and thought I'll make a proposal for that too. After completing my proposal for glTF integration I started my research on this one too.
+
+I started writing the proposal early so that I could get my proposal reviewed at least once with the project mentors. I almost forgot the fact that I had my end-term examinations at hand (Somehow I managed to pass all of my courses), I kept contributing to the project till the end of April 2022.
+
+Code contributions:
+
+1. [https://github.com/fury-gl/fury/pull/520]
+2. [https://github.com/fury-gl/fury/pull/525]
+3. [https://github.com/fury-gl/fury/pull/533]
+4. [https://github.com/fury-gl/fury/pull/547]
+5. [https://github.com/fury-gl/fury/pull/556]
+6. [https://github.com/fury-gl/fury/pull/559]
+
+The Day
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
too short, should be the same size as the title. the doc generation creates multiple warnings.